### PR TITLE
Gic bugfix

### PR DESCRIPTION
**Pull request by i-nevo**

### CODE REVIEW

File: core/drivers/gic.c [hunk 1]
```diff
@@ -66,9 +66,7 @@
 #define ITARGETSR_FIELD_BITS	8
 #define ITARGETSR_FIELD_MASK	0xff
 
-/* Maximum number of interrups a GIC can support */
-#define GIC_MAX_INTS		1020
-
+#define GICD_TYPER_IT_LINES_NO_MASK	0x1f
```
Comment (jenswi-linaro) [inline core/drivers/gic.c:69]: s/NO/NUM/

**Review by jenswi-linaro — COMMENTED** [review]
Please change the title of the commit message to
"drivers: gic: prevent accessing unimplemented GIC registers"

With my comments addressed:
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jenswi-linaro** [comment]
Please remove the `.` at the end of the commit title.

**Review by etienne-lms — APPROVED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@foss.st.com>`

**Comment by etienne-lms** [comment]
nitpicking: in commit message: s/Check/check/

**Comment by etienne-lms** [comment]
@jforissier, could you trigger CI test for this P-R?

**Pull request merged**